RadixWallet/Clients/TransportProfileClient/TransportProfileClient+Interface.swift

```diff
@@ -0,0 +1,26 @@
+import Sargon
+
+public typealias DeviceID = UUID
+
+// MARK: - TransportProfileClient
+public struct TransportProfileClient: Sendable {
+	public var importProfile: ImportProfile
+	public var profileForExport: ProfileForExport
+	public var didExportProfile: DidExportProfile
+
+	public init(
+		importProfile: @escaping ImportProfile,
+		profileForExport: @escaping ProfileForExport,
+		didExportProfile: @escaping DidExportProfile
+	) {
+		self.importProfile = importProfile
+		self.profileForExport = profileForExport
+		self.didExportProfile = didExportProfile
+	}
+}
+
+extension TransportProfileClient {
+	public typealias ImportProfile = @Sendable (Profile, Set<FactorSourceIDFromHash>, Bool) async throws -> Void
```

Comment:
if you write `, _ containsP2PLinks: Bool)` in the typealias one knows what the Bool is just by looking at the typealias, or write a comment? :)